src/fixate/drivers/dmm/fluke_8846a.py

```diff
@@ -341,3 +342,11 @@ def analog_filter(self, bandwidth=None):
         elif any(x in self._mode for x in ["dc", "res"]):
             self._write(self._filters[self.mode] + ":FILT:STAT ON")
         pass
+
+    def get_serial(self):
```

Comment:
I think we should either parse out the serial number or rename this to get_identity.
Possibly, rename this to get identity, the implement get_serial using this function.

Reply:
I originally was doing some string parsing to get the serial number out, but realised that including everything meant that we could keep the other parameters such as model number, while still letting us filter through them. 
I'll rename it to get_identity